pages/project-versions/submit/content/index.js

```diff
@@ -1,15 +1,18 @@
 module.exports = {
-  title: 'Send application',
+  title: 'Send {{model.type}}',
   warning: 'Project licence applications need to be reviewed by an AWERB and endorsed by the primary establishment\'s PEL holder before a licence can be granted.',
   fields: {
     authority: {
-      label: 'Does this application have the endorsement of your primary establishment\'s PEL holder?'
+      label: 'Does this {{model.type}} have the endorsement of your primary establishment\'s PEL holder?'
     },
     awerb: {
       label: 'Has this application been reviewed by the AWERB of each relevant establishment?'
     },
     ready: {
       label: 'Is this version of your application ready for an inspector to approve?'
+    },
+    comment: {
+      label: 'Why are you making this amendment?'
     }
   },
   errors: {
```

Comment:
you haven't added an error message for when the comment is missing - this will break the page if validation fails I think

Reply:
done